chipy_org/apps/meetings/admin.py

```diff
@@ -23,15 +23,15 @@ class TopicInline(admin.StackedInline):
 
 
 class TopicAdmin(admin.ModelAdmin):
-    list_display = ('id', 'approved', 'title', 'experience_level', 'meeting', 'created')
+    list_display = ('id', 'approved', 'title', 'experience_level', 'get_presenters', 'meeting', 'created')
     readonly_fields = ['get_presenters', 'modified', 'created', ]
     list_filter = ['approved', 'experience_level']
     search_fields = ['title']
     filter_horizontal = ['presentors']
 
     def get_presenters(self, obj):
         return format_html(" &bull; ".join(
-            [f"<a href='{reverse('admin:meetings_presentor_change', args=[p.id])}'>{p}</a>"
+            [f"<a href='{reverse('admin:meetings_presentor_change', args=[p.id])}'>{p.name}</a>"
```

Comment:
This is so we limit to just seeing the name and not the markup that's part of __str__